cscs-checks/microbenchmarks/stream/stream.py

```diff
@@ -11,8 +11,7 @@ def __init__(self):
         self.exclusive_access = True
         # All available systems are supported
         self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc',
-                              'kesch:cn', 'kesch:pn', 'leone:normal',
-                              'monch:compute']
+                              'kesch:cn', 'kesch:pn', 'leone:normal']
```

Comment:
we are removing the system, but not the performance reference number.
Is there any reason why? Do we want to keep for future reference?

Reply:
I thought we wanted them for reference, don't we?